.github/workflows/main.yaml

```diff
@@ -0,0 +1,16 @@
+name: Build and test
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Curious if this is just a hint about which platform its tested on?

Reply:
It's more than a hint: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on